elasticgraph-schema_artifacts/spec/unit/elastic_graph/schema_artifacts/runtime_metadata/enum_spec.rb

```diff
@@ -29,12 +29,7 @@ module Enum
           it "builds from a minimal hash" do
             enum_value = Enum::Value.from_hash({})
 
-            expect(enum_value).to eq Enum::Value.new(
-              sort_field: nil,
-              datastore_value: nil,
-              datastore_abbreviation: nil,
-              alternate_original_name: nil
-            )
+            expect(enum_value).to eq enum_value_with
```

Comment:
I actually prefer using `Enum::Value.new` in this case because this is directly testing the `Enum::Value` class. In direct tests of a class, I like being forced to fully specify all fields (since the class itself is what's being tested). In tests focused on other code which happen to use an `Enum::Value` I like using `enum_value_with` so that those tests can omit the details that don't matter to the test.